src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private final SimpleDateFormat dateFormatter = new SimpleDateFormat("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        try {
+            final Date dateFromFormatted = dateFormatter.parse(dateFrom);
+            final Date dateToFormatted = dateFormatter.parse(dateTo);
```

Comment:
```suggestion
           LocalDate formattedDateFrom = dateFormatter.parse(dateFrom);
           LocalDate formattedDateTo= dateFormatter.parse(dateTo);
```
<img width="494" alt="Screen Shot 2022-08-18 at 9 17 26 PM" src="https://user-images.githubusercontent.com/90692308/185465946-0be4a6b0-9009-4864-a5d7-bdef9af62c09.png">